app/models/user.rb

```diff
@@ -156,6 +157,10 @@ def self.skip_mx_check?
     Rails.env.local?
   end
 
+  def set_role_id
+    self.role_id ||= -99
+  end
```

Comment:
Is this necessary, considering that there's a default on the database level?

Reply:
It is necessary for existing records.

Reply:
I believe that the existing records are still going to use the default from the database, aren't they?